main.go

```diff
@@ -16,7 +16,6 @@ import (
 	"github.com/bitrise-io/go-utils/pathutil"
 	"github.com/bitrise-io/go-utils/retry"
 	"github.com/bitrise-tools/go-steputils/input"
-	"github.com/kballard/go-shellquote"
```

Comment:
any idea why this was removed?

Reply:
Oops! That was definitely an accident. I'll push an update that adds it back. Good catch!